realtime_tools/CMakeLists.txt

```diff
@@ -44,7 +44,7 @@ target_include_directories(realtime_tools PUBLIC
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include/realtime_tools>
 )
-target_link_libraries(realtime_tools PUBLIC rclcpp::rclcpp rclcpp_action::rclcpp_action rcpputils::rcpputils Threads::Threads Boost::boost)
```

Comment:
As by investigation of @saikishor, adding Boost::boost here does not link in the traditional sense but ensures that the include directories are setup